CHANGELOG.md

```diff
@@ -109,6 +109,7 @@
 - PR #2809 Add hash_df and group_split dispatch functions for dask
 - PR #2843 Parquet reader: fix skip_rows when not aligned with page or row_group boundaries
 - PR #2851 Deleted existing dask-cudf/record.txt
+- PR #2708 Fixed `cumsum()` and `cumprod()` on boolean series.
```

Comment:
Use PR #2852 instead

Reply:
sorry my bad :(

Reply:
No worries, It was confusing for me at first. Catch 22, since you have to create a PR before you can get the number, but the checks wont pass without the pr number in the change log. :man_shrugging: